app/controllers/application_controller/performance.rb

```diff
@@ -702,6 +702,15 @@ def perf_gen_data_before_wait
                              from_dt,
                              to_dt,
                              interval_type]
+      elsif @perf_record.kind_of?(MiddlewareServer)
```

Comment:
I saw the realtime section is on line 730 below, so what is this report standing for?